.github/workflows/run-cron.yml

```diff
@@ -1,6 +1,10 @@
 name: run-cron
 
-on: workflow_dispatch
+on:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
For future reference, `00:00` UTC is `8:00pm EDT`/`5:00pm PDT`:

![Screenshot 2022-08-01 at 10 26 56 PM](https://user-images.githubusercontent.com/13009507/182278506-c50bf5d8-c340-44dc-916b-96fb4b44b56c.jpg)

See https://crontab.guru/#0_0_*_*_0